src/svn.py

```diff
@@ -0,0 +1,26 @@
+import re
+
+from bs4 import BeautifulSoup
+from common import dates, http, releasedata
+
+# https://regex101.com/r/CwvyT3/2 only non beta versions
+VERSION_PATTERN = re.compile(r"^(?:Subversion\s)(\d.\d+\.\d+$)")
+# https://regex101.com/r/GsimYd/1
+DATE_PATTERN = re.compile(r"^(?:\s\()(\w+,\s\d{1,2}\s\w+\s\d{4})")
+URL = "https://subversion.apache.org/docs/release-notes/release-history.html"
+
+with releasedata.ProductData("svn") as product_data:
```

Comment:
```suggestion
with releasedata.ProductData("apache-subversion") as product_data:
```

This and the script name must match the name of the markdown file in https://github.com/endoflife-date/endoflife.date/pull/5567.

Reply:
Done.